great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -352,3 +352,73 @@ def _instantiate_datasource_from_config_and_update_project_config(
                 raise e
 
         return datasource
+
+    def add_checkpoint(
+        self,
+        name: str,
+        config_version: Optional[Union[int, float]] = None,
```

Comment:
Require this level of verbosity to maintain parity with `BaseDataContext` - can we break off from that pattern here or do we need consistency?